### DISCUSSION TITLE
How to train new token

### DISCUSSION THREAD

**Discussion by fclearner** [QwenLM/Qwen3#1091] [Q&A]
I would like to add a new token on Qwen, using the reserved unused token ID. I have tried both LoRA fine-tuning and full fine-tuning of the embeddings, but it seems that the loss for the new token converges poorly and the accuracy remains at 0. Are there any specific techniques required for fine-tuning newly added tokens?

**Comment by zkailinzhang** [comment]
mark

**Reply by J123ingjing** [reply]
Hello！ Do you have any idea about it？I'm happy if you share it!
